Library/Homebrew/test/spec_helper.rb

```diff
@@ -41,6 +41,9 @@
 RSpec.configure do |config|
   config.order = :random
 
+  config.filter_run_when_matching :focus
+  config.run_all_when_everything_filtered = true
```

Comment:
This second line isn't needed, and can lead to some confusing behaviour, if I'm reading http://rspec.info/blog/2016/07/rspec-3-5-has-been-released/ right